cypari2/paridecl.pxd

```diff
@@ -1,4 +1,3 @@
-# distutils: libraries = gmp pari
 """
```

Comment:
Why remove these? I think they are useful

Reply:
argh. Indeed, it is for people using cython with cypari...

Reply:
better with de8b0cb (forced push)